R/angles.R

```diff
@@ -63,6 +67,7 @@ calculate.angles.radial <- function(
                     v = v,
                     dx.scale = median(tree$length1)
                     );
+                # angles[tree$tip %in% child.ids] <- start.angle + angles[tree$tip %in% child.ids];
```

Comment:
Same question here. We're sure that `split.equal.x.dist` accounts for the parent angle properly?

Reply:
This is from my naive try to just add `start.angle` to the dendrogram, but it doesn't exist prior to this PR.